proc/eval.go

```diff
@@ -432,16 +432,23 @@ func (scope *EvalScope) evalIdent(node *ast.Ident) (*Variable, error) {
 	// try to interpret this as a local variable
 	v, err := scope.extractVarInfo(node.Name)
 	if err != nil {
-		// if it's not a local variable then it could be a package variable w/o explicit package name
 		origErr := err
-		_, _, fn := scope.Thread.dbp.PCToLine(scope.PC)
-		if fn != nil {
-			if v, err := scope.packageVarAddr(fn.PackageName() + "." + node.Name); err == nil {
-				v.Name = node.Name
-				return v, nil
+		// workaround: sometimes go inserts an entry for '&varname' instead of varname
```

Comment:
Is there an issue in the Go tracker you can point to here? If the issue gets fixed it would be nice to have a link in this comment so we can discuss removing the workaround once this gets fixed.